### PR TITLE
bump aws and simplify credentials provider

### PR DESCRIPTION
**Pull request by akash1810**
Just keeping things up to date.

Using a simpler CredentialsProvider as we don't need all the items in the current chain.

### CODE REVIEW

File: common-lib/src/main/scala/com/gu/workflow/util/AWS.scala [hunk 1]
```diff
@@ -1,31 +1,39 @@
...
   lazy val region: Region = Region getRegion Regions.EU_WEST_1
 
-  lazy val EC2Client = AmazonEC2ClientBuilder.standard.withRegion(region.getName).build
-  lazy val CloudWatch = AmazonCloudWatchAsyncClientBuilder.standard.withRegion(region.getName).build
-  lazy val DynamoDb = AmazonDynamoDBClientBuilder.standard.withRegion(region.getName).withCredentials(
-    new AWSCredentialsProviderChain(
-      new EnvironmentVariableCredentialsProvider(),
-      InstanceProfileCredentialsProvider.getInstance(),
-      new ProfileCredentialsProvider("workflow"),
-      new DefaultAWSCredentialsProviderChain
```
Comment (akash1810) [inline common-lib/src/main/scala/com/gu/workflow/util/AWS.scala:26]: We're [using an InstanceProfile](https://github.com/guardian/editorial-tools-platform/blob/master/cloudformation/workflow-account/workflow/workflow-frontend.yaml#L326), so provider chain [can become a lot simpler](https://github.com/guardian/workflow-frontend/pull/216/files#diff-2b19a54750b96913359e3934504c2b0dR15).

Comment (AWare) [reply common-lib/src/main/scala/com/gu/workflow/util/AWS.scala:26]: What about running locally? Shouldn't that still use the profile provider?

Comment (akash1810) [reply common-lib/src/main/scala/com/gu/workflow/util/AWS.scala:26]: > What about running locally? Shouldn't that still use the profile provider?

Yep - https://github.com/guardian/workflow-frontend/pull/216/files#diff-2b19a54750b96913359e3934504c2b0dR16

Comment (akash1810) [reply common-lib/src/main/scala/com/gu/workflow/util/AWS.scala:26]: GH doesn't render that link very obviously. An image of the diff maybe?

image

Comment (AWare) [reply common-lib/src/main/scala/com/gu/workflow/util/AWS.scala:26]: ohh, I see. I thought that AWS was _AWS_

**Pull request merged**